pages/api/index.js

```diff
@@ -10,6 +12,14 @@ function API({url, content, locale}) {
       locale={locale}
       contributors={content.contributors}
     >
+      <Head>
+        {twitterCard({
+          card: 'summary',
+          title: content.title,
```

Comment:
I don't have a strong opinion either way, so long as the name `glamorous` is obvious every time. Maybe the first way is better.

It should match the page `title`, so whichever is chosen maybe you can update that as well?